scenarios/kube/cpu-hog.yml

```diff
@@ -0,0 +1,9 @@
+duration: 60
+workers: '' # leave it empty '' node cpu auto-detection
+hog-type: cpu
+image: quay.io/krkn-chaos/krkn-hog
+namespace: default
+cpu-load-percentage: 90
+cpu-method: all
+node-selector: "node-role.kubernetes.io/worker="
```

Comment:
We might want to expose an option to pick the number of nodes to pick matching the node-selector. This way user can choose to pick 1 or few nodes to target instead of all nodes matching the selector. Thoughts?